src/sentry/api/endpoints/organization_member_team_details.py

```diff
@@ -292,4 +292,4 @@ def delete(
             )
             omt.delete()
 
-        return Response(status=200)
+        return Response(serialize(team, request.user, TeamWithProjectsSerializer()), status=200)
```

Comment:
`TeamWithProjectsSerializer()` is tagged as deprecated? Is there another option we could use?

The response also leaks the internal ID and would get flagged as an IDOR vulnerability. Is there specific subset of the data the frontend needs that can be returned?

https://github.com/getsentry/sentry/blob/48c55acdbf09b408160e0386483380b72cb0aec9/src/sentry/api/serializers/models/team.py#L254-L258

Reply:
how's the TeamSerializer sound? We do not need the projects.